pkg/postgresql/client.go

```diff
@@ -89,7 +90,7 @@ func NewClient(cfg *Config) *Client {
 		return sql.Open("postgres", connStr)
 	})
 
-	log.Info("msg", connStr)
+	log.Info("msg", regexp.MustCompile("password='(.+)'").ReplaceAllLiteralString(connStr, "password='****'"))
```

Comment:
Can we change the regex here to a non-greedy one just to be safe?

Reply:
I might also recommend that the `regexp.MustCompile` part be moved into the `var` above so it only makes one regexp

Reply:
@antekresic I am making it lazy but I am force pushing the change because 2 commits for a single line change seems unnecessary.
@RobAtticus this is only called once, so moving `regexp.MustCompile` to `var` above wouldn't make much of a difference.